entviz.go

```diff
@@ -83,34 +83,33 @@ func generateHTML(g *gen.Graph) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
-func generateGoFile() ([]byte, error) {
-	var b bytes.Buffer
-	if err := entviztmpl.Execute(&b, nil); err != nil {
-		return nil, err
-	}
-	return b.Bytes(), nil
+// VisualizeSchema is an ent hook that generates a static html page that visualizes the schema graph.
+func VisualizeSchema(next gen.Generator) gen.Generator {
+	return gen.GenerateFunc(func(g *gen.Graph) error {
+		buf, err := generateHTML(g)
+		if err != nil {
+			return err
+		}
+		path := filepath.Join(g.Config.Target, "schema-viz.html")
```

Comment:
Not sure what is the `cwd` in this case, but using `filepath.Join` with the target dir makes sure the file will b written to the desired package.